src/device-api/middleware.ts

```diff
@@ -0,0 +1,141 @@
+import * as morgan from 'morgan';
+import * as _ from 'lodash';
+
+import * as config from '../config';
+import * as apiKeys from '../lib/api-keys';
```

Comment:
Should we move the `lib/api-keys` module to `./device-api`?

Reply:
I wanted to, but saw that they are used in some other places, notably in [reporting initial state in device-state.ts](https://github.com/balena-os/balena-supervisor/blob/234a46c3cb163af959dce7969dfa09db78156560/src/device-state.ts#L249-L263) and [when generating a SV API key when the compose label is present](https://github.com/balena-os/balena-supervisor/blob/104a8006fbab4bad0f2da923eade0e64002dd59f/src/compose/utils.ts#L374-L377). Is the purpose of this move to indicate that "api-keys.ts belongs to device-api but can be used elsewhere"? If so then I can move it, no problem.

Reply:
Moved with https://github.com/balena-os/balena-supervisor/commit/5714830fceef45714312d08aeb5229b8324eb2c7